helm/fiftyone-teams-app/README.md

```diff
@@ -0,0 +1,725 @@
+
+- [Initial Installation vs. Upgrades](#initial-installation-vs-upgrades)
+- [FiftyOne Features](#fiftyone-features)
+  - [FiftyOne Teams Authenticated API](#fiftyone-teams-authenticated-api)
+  - [FiftyOne Teams Plugins](#fiftyone-teams-plugins)
+  - [Storage Credentials and `FIFTYONE_ENCRYPTION_KEY`](#storage-credentials-and-fiftyone_encryption_key)
+  - [Proxies](#proxies)
+    - [Text Similarity](#text-similarity)
+- [Upgrading From Previous Versions](#upgrading-from-previous-versions)
+  - [From Early Adopter Versions (Versions less than 1.0)](#from-early-adopter-versions-versions-less-than-10)
+  - [From Before FiftyOne Teams Version 1.1.0](#from-before-fiftyone-teams-version-110)
+  - [From FiftyOne Teams Version 1.1.0 and later](#from-fiftyone-teams-version-110-and-later)
+- [Launch FiftyOne Teams](#launch-fiftyone-teams)
+  - [Simple Example](#simple-example)
+  - [A Full Deployment Example on GKE](#a-full-deployment-example-on-gke)
+    - [Download the Example Configuration Files](#download-the-example-configuration-files)
+    - [Create the Necessary Helm Repos](#create-the-necessary-helm-repos)
+  - [Install and Configure cert-manager](#install-and-configure-cert-manager)
+    - [Create a ClusterIssuer](#create-a-clusterissuer)
+    - [Install and Configure MongoDB](#install-and-configure-mongodb)
+    - [Obtain a Global Static IP Address and Configure a DNS Entry](#obtain-a-global-static-ip-address-and-configure-a-dns-entry)
+    - [Set up http to https Forwarding](#set-up-http-to-https-forwarding)
+    - [Install FiftyOne Teams App](#install-fiftyone-teams-app)
+    - [Installation Complete](#installation-complete)
+
+<!-- tocstop -->
+
+We publish container images to these Docker Hub repositories
+
+- `voxel51/fiftyone-app`
+- `voxel51/fiftyone-app-gpt`
+- `voxel51/fiftyone-app-torch`
+- `voxel51/fiftyone-teams-api`
+- `voxel51/fiftyone-teams-app`
+
+For Docker Hub credentials, please contact your Voxel51 support team.
+
+## Initial Installation vs. Upgrades
+
+Upgrades are more frequent than new installations.
+Thus, the chart's default behavior supports
+upgrades and the `values.yaml` contains
+
+```yaml
+appSettings:
+  env:
+    FIFTYONE_DATABASE_ADMIN: false
+```
+
+When performing an initial installation,
+in your `values.yaml`, set
+
+```yaml
+appSettings:
+  env:
+    FIFTYONE_DATABASE_ADMIN: true
+```
+
+After the initial installation, either comment this
+environment variable or change the value to false.
+
+When performing an upgrade, please review our
+[Upgrading From Previous Versions](#upgrading-from-previous-versions)
+
+## FiftyOne Features
+
+Consider if you will require these settings for your deployment.
+
+### FiftyOne Teams Authenticated API
+
+FiftyOne Teams v1.3 introduced the capability to connect FiftyOne Teams SDKs
+through the FiftyOne Teams API (instead of direct MongoDB connection).
+
+To enable the FiftyOne Teams Authenticated API,
+[expose the FiftyOne Teams API endpoint](docs/expose-teams-api.md)
```

Comment:
```suggestion
[expose the FiftyOne Teams API endpoint](../docs/expose-teams-api.md)
```

Reply:
Moving the README, but not the docs directory, means only part of the docs will ship with the helm chart.  Do you need to move the `docs` directory here too?

Reply:
If you think we should include it, we can. Let me know what you'd like.

Reply:
well, we probably need to either update the links to be fully-qualified pointing at helm.fiftyone.ai or bundle them - otherwise the links won't work...

Either way, doesn't really matter to me

Reply:
I'll update the links to be :

* https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/docs/expose-teams-api.md
* https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/docs/plugins-storage.md

That way, whether viewed in GH or in the chart's README.md, they always work.